security/docker/app.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
```

Comment:
Could you add comment about what this file does? Thanks.

Reply:
Done